soaks/bin/soak_one.sh

```diff
@@ -21,11 +22,16 @@ display_usage() {
     echo "  --cpus: the total number of CPUs to dedicate to the soak minikube, default 7"
     echo "  --memory: the total amount of memory dedicate to the soak minikube, default 8g"
     echo "  --vector-cpus: the total number of CPUs to give to soaked vector"
-    echo "  --warmup-seconds: the total number seconds to pause waiting for vector to warm up"
+    echo "  --warmup-seconds: the total number seconds to pause waiting for vector to warm up, default 30"
+    echo "  --total-samples: the total number of samples to take from vector, default 200"
+    echo "  --replicas: the total number of replica experiments to run, default 3"
     echo ""
 }
 
 BUILD_IMAGE="true"
+REPLICAS=3
```

Comment:
We also have replicas set by the ci workflow, what's the difference between the two?

Reply:
This is the default setting. If we didn't have `REPLICAS` here the program would fail to run if the user didn't supply a `--replicas` flag. I'm increasingly inclined to merge `soak_one` up into `soaks` but that's for a later PR.